src/main/java/org/symphonyoss/symphony/messageml/elements/TextField.java

```diff
@@ -61,9 +61,10 @@ public void validate() throws InvalidInputException {
       assertAttributeValue(MASKED_ATTR, VALID_BOOLEAN_VALUES);
     }
 
-    validateMinAndMaxLengths();
     assertAttributeNotBlank(NAME_ATTR);
+    // guarantees that we will only have one children and its the expected type: TextNode
```

Comment:
The method below only guarantees the type of children, not the number of it. I don' t think we should leave this comment here though